omnigibson/utils/usd_utils.py

```diff
@@ -697,10 +697,46 @@ def reset(cls):
         cls.MODIFIED_PRIMS = set()
 
 
+class PoseAPI:
+    DIRTY = True
```

Comment:
let's call this `VALID` instead, and the following two functions `validate` and `invalidate`